multinet/db.py

```diff
@@ -3,10 +3,10 @@
 
 from arango import ArangoClient
 from arango.database import StandardDatabase, StandardCollection
-from arango.exceptions import DatabaseCreateError
+from arango.exceptions import DatabaseCreateError, EdgeDefinitionCreateError
```

Comment:
It looks like this is causing a typechecking error; the fix is to update the manual definitions of arango types in the top level `mypy_stubs` directory.